helm/flux-app/templates/source.yaml

```diff
@@ -57,6 +57,10 @@ metadata:
 spec:
   interval: "{{ .interval }}"
   path: "{{ .path }}"
+{{- if .postBuild }}
+  postBuild:
```

Comment:
Can we document somewhere that this option exists? Maybe a commented out block in the `values.yaml`?

Reply:
+1

Reply:
Sure, let's put it into both, the `values.yaml` and `Readme`

Reply:
@MarcelMue what about now?